book-library/script.js

```diff
@@ -52,15 +45,18 @@ function Book(title, author, pages, check) {
 
 function render() {
   let table = document.getElementById("display");
+  let tbody = table.getElementsByTagName('tbody')[0];
+
   let rowsNumber = table.rows.length;
-  //delete old table
-  for (let n = rowsNumber - 1; n > 0; n-- {
+  // delete old table
+  for (let n = rowsNumber - 1; n > 0; n--) {
     table.deleteRow(n);
   }
-  //insert updated row and cells
+
+  // insert updated row and cells
   let length = myLibrary.length;
```

Comment:
Use const where appropriate. When setting a variable using the let keyword we are signalling that we will change the value at a later point. if you intend to keep the value the same use "const"